js/src/rive.ts

```diff
@@ -1452,6 +1456,65 @@ export class Rive {
     }
   }
 
+  public handleTouchEvents() {
```

Comment:
What if we called this something like "registerTouchEvents"?

"handleTouchEvents" made me wonder if it was intended to be some sort of main loop or something.